### PR TITLE
[Feature] #19, #20 : Add ToolDictionaryDetailView UI

### LINKED ISSUE #19: [Feature] Construct ToolDictionaryDetailView's scrollView UI
# 유저는 장비군을 다시 선택할 수 있다.
- 장비군을 스크롤 해서 원하는 장비군을 볼 수 있다.

## 🤔 Completion conditions

- [ ] Construct ToolDictionaryDetailView's scrollView UI like the design of a sketch ( low-fi )

## 👀 Related issues

- Related issue1 #20 
- Related issue2 #21

### LINKED ISSUE #20: [Feature] Construct ToolDictionaryDetailView's Detail Tools UI
# 유저는 장비군을 다시 선택할 수 있다.
- 유저는 장비군에 해당하는 상세 장비들을 확인할 수 있다.

## 🤔 Completion conditions

- [ ] Construct ToolDictionaryDetailView's Detail Tools UI like the design of a sketch ( low-fi )

## 👀 Related issues

- Related issue1 #19 
- Related issue2 #21

### PR DESCRIPTION
**Pull request by rriver2**
# Issue Number
🔒 Close #19 
🔒 Close #20 

## Changes

- Changed files
    - campair/campair/View/ToolDictionaryDetailView.swift

## New features

- Create the ToolDictionaryDetailView's SearchBar UI (ScrollView , show detail tools)

## Review points

- I don't know if I split the view in a good way.

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Review by sustainable-git — CHANGES_REQUESTED** [review]
There is no SearchBar UI...

File: campair/campair/View/ToolDictionaryDetailView.swift [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            Divider()
+        }
+        .padding(.bottom, 20)
+    }
+}
+
+struct SelectingToolView: View {
+    var body: some View {
+        Button {
+
```
Comment (sustainable-git) [inline campair/campair/View/ToolDictionaryDetailView.swift:65]: use `// todo` here

File: campair/campair/View/ToolDictionaryDetailView.swift [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+}
+
+struct SelectingToolView: View {
+    var body: some View {
+        Button {
+            
+        } label: {
+            VStack(alignment: .leading, spacing: 0){
+                HStack( spacing: 30){
```
Comment (sustainable-git) [inline campair/campair/View/ToolDictionaryDetailView.swift:68]:
```suggestion
                HStack(spacing: 30) {
```
check indent! :)

File: campair/campair/View/ToolDictionaryDetailView.swift [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+struct SelectingToolDictionaryView: View {
+    @Binding var selectedToolNumber : Int
+    var body: some View {
+        ZStack(alignment: .bottom){
+            ScrollView(.horizontal, showsIndicators: false){
+                HStack(alignment: .top){
+                    ForEach(0..<16){ index in
+                        Button(action: {
```
Comment (sustainable-git) [inline campair/campair/View/ToolDictionaryDetailView.swift:35]:
```suggestion
                        Button {
                            self.selectedToolNumber = index
                        } label: {
```
Keep closure expressions simple :)

File: campair/campair/View/ToolDictionaryDetailView.swift (hunk 1 above)
Comment (rriver2) [reply campair/campair/View/ToolDictionaryDetailView.swift:65]: Good !

File: campair/campair/View/ToolDictionaryDetailView.swift (hunk 2 above)
Comment (rriver2) [reply campair/campair/View/ToolDictionaryDetailView.swift:68]: good

File: campair/campair/View/ToolDictionaryDetailView.swift (hunk 3 above)
Comment (rriver2) [reply campair/campair/View/ToolDictionaryDetailView.swift:35]: thank you ~

**Pull request merged**